assets/loadtest/helm/database-agent/values.yaml

```
@@ -0,0 +1,29 @@
+replicaCount: 1
+agentsPerPod: 10
+proxyServer: ""
+authServer: ""
+
+minReadySeconds: 0
+
+image:
+  repository: public.ecr.aws/gravitational/teleport
+  pullPolicy: IfNotPresent
+  tag: ""
+
+serviceAccount:
+  create: true
+  roleARN:
+
+joinParams:
+  method: token
+  token_name: qwertyuiop
```

the camel_case name is inconsistent. Also by default can we just point to token file even if we will not have mounted secret ?
```suggestion
  authToken:  /etc/teleport-secrets/auth-token
```